dart/collision/fcl/FCLTypes.hpp

```diff
@@ -55,25 +79,97 @@ template <class T> using fcl_shared_ptr = boost::shared_ptr<T>;
 template <class T> using fcl_weak_ptr = boost::weak_ptr<T>;
 #endif
 
-namespace dart {
-namespace collision {
+namespace dart
+{
+namespace fcl
+{
+#if FCL_VERSION_AT_LEAST(0,6,0)
+// Geometric fundamentals
+typedef ::fcl::Vector3<double> Vector3;
+typedef ::fcl::Matrix3<double> Matrix3;
+typedef ::fcl::Transform3<double> Transform3;
+// Geometric primitives
+typedef ::fcl::Box<double> Box;
+typedef ::fcl::Cylinder<double> Cylinder;
+typedef ::fcl::Ellipsoid<double> Ellipsoid;
+typedef ::fcl::Halfspace<double> Halfspace;
+typedef ::fcl::Sphere<double> Sphere;
+// Collision objects
+typedef ::fcl::CollisionObject<double> CollisionObject;
+typedef ::fcl::CollisionGeometry<double> CollisionGeometry;
+typedef ::fcl::DynamicAABBTreeCollisionManager<double> DynamicAABBTreeCollisionManager;
+typedef ::fcl::OBBRSS<double> OBBRSS;
+typedef ::fcl::CollisionRequest<double> CollisionRequest;
+typedef ::fcl::CollisionResult<double> CollisionResult;
+typedef ::fcl::DistanceRequest<double> DistanceRequest;
+typedef ::fcl::DistanceResult<double> DistanceResult;
+typedef ::fcl::Contact<double> Contact;
+#else
+// Geometric fundamentals
+typedef ::fcl::Vec3f Vector3;
+typedef ::fcl::Matrix3f Matrix3;
+typedef ::fcl::Transform3f Transform3;
+// Geometric primitives
+typedef ::fcl::Box Box;
+typedef ::fcl::Cylinder Cylinder;
+typedef ::fcl::Halfspace Halfspace;
+typedef ::fcl::Sphere Sphere;
+// Collision objects
+typedef ::fcl::CollisionObject CollisionObject;
+typedef ::fcl::CollisionGeometry CollisionGeometry;
+typedef ::fcl::DynamicAABBTreeCollisionManager DynamicAABBTreeCollisionManager;
+typedef ::fcl::OBBRSS OBBRSS;
+typedef ::fcl::CollisionRequest CollisionRequest;
+typedef ::fcl::CollisionResult CollisionResult;
+typedef ::fcl::DistanceRequest DistanceRequest;
+typedef ::fcl::DistanceResult DistanceResult;
+typedef ::fcl::Contact Contact;
+#endif
+
+#if FCL_VERSION_AT_LEAST(0,4,0) && !FCL_VERSION_AT_LEAST(0,6,0)
+typedef ::fcl::Ellipsoid Ellipsoid;
+#endif
+
+using ::fcl::GJKSolverType;
+
+template<typename BV>
+using BVHModel = ::fcl::BVHModel<BV>;
```

Comment:
I believe there were no changes for `BVHModel` across 0.5 and 0.6. Could you remove this type aliasing?